### PR TITLE
[NUI] Add GetOriginalImageSize() considering rotation

### PR DESCRIPTION
**Pull request by tscholb**
### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI/src/public/Images/ImageLoading.cs [hunk 1]
```diff
@@ -420,7 +420,25 @@ public static Size2D GetClosestImageSize(string filename)
...
+        /// </summary>
+        /// <param name="filename">The name of the image.</param>
+        /// <param name="orientationCorrection">Reorient the image to respect any orientation metadata in its header.</param>
+        /// <returns>Dimension of the original image.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        /// This will be released at Tizen.NET API Level 9. Therefore, currently this would be used as an in-house API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static Size2D GetOriginalImageSize(string filename, bool orientationCorrection )
+        {
+            var val = new Uint16Pair(Interop.ImageLoading.GetOriginalImageSize_SWIG_1(filename,orientationCorrection), true);
```
Comment (huiyueun) [inline src/Tizen.NUI/src/public/Images/ImageLoading.cs:441]: Please follow C# Coding rule

```csharp
(filename, orientationCorrection)
```

File: src/Tizen.NUI/src/public/BaseComponents/ImageView.cs [hunk 2]
```diff
@@ -1244,7 +1244,7 @@ private void UpdateImage(int key, PropertyValue value)
             {
                 if (_resourceUrl != null)
                 {
-                  Size2D imageSize = ImageLoading.GetOriginalImageSize(_resourceUrl);
+                  Size2D imageSize = ImageLoading.GetOriginalImageSize(_resourceUrl,true);
```
Comment (huiyueun) [inline src/Tizen.NUI/src/public/BaseComponents/ImageView.cs:1247]: Please add space.
```csharp
(_resourceUrl, true)
```

File: src/Tizen.NUI/src/public/Images/ImageLoading.cs [hunk 3]
```diff
@@ -420,7 +420,25 @@ public static Size2D GetClosestImageSize(string filename)
...
+        /// <summary>
+        /// Get the size of an original image consider rotation
+        /// </summary>
+        /// <param name="filename">The name of the image.</param>
+        /// <param name="orientationCorrection">Reorient the image to respect any orientation metadata in its header.</param>
+        /// <returns>Dimension of the original image.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        /// This will be released at Tizen.NET API Level 9. Therefore, currently this would be used as an in-house API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static Size2D GetOriginalImageSize(string filename, bool orientationCorrection )
```
Comment (huiyueun) [inline src/Tizen.NUI/src/public/Images/ImageLoading.cs:439]: Remove space
```csharp
(string filename, bool orientationCorrection)
```

File: src/Tizen.NUI/src/public/Images/ImageLoading.cs [hunk 4]
```diff
@@ -420,7 +420,25 @@ public static Size2D GetClosestImageSize(string filename)
         [EditorBrowsable(EditorBrowsableState.Never)]
         public static Size2D GetOriginalImageSize(string filename)
```
Comment (huiyueun) [inline src/Tizen.NUI/src/public/Images/ImageLoading.cs:421]: How about providing only one API?
GetOriginalImageSize is hidden API, so we can modify it.
'''csharp
public static Size2D GetOriginalImageSize(string filename, bool orientationCorrection = false)
'''

**Review by huiyueun — APPROVED** [review]

**Review by hyunjushin — COMMENTED** [review]
please fix url related warning also. you can make separate patch for them.

File: src/Tizen.NUI/src/public/Images/ImageLoading.cs (hunk 4 above)
Comment (hyunjushin) [reply src/Tizen.NUI/src/public/Images/ImageLoading.cs]: I agree

**Review by adeelkazmi — APPROVED** [review]

**Comment by rabbitfor** [comment]
@tscholb Please merge this patch yourself after check related dali-csharp-binder patch is ready.

**Comment by huiyueun** [comment]
@tscholb Could you resolve conflicts and merge it?

**Pull request closed without merging**